scripts/build.sh

```diff
@@ -164,6 +164,17 @@ cortxfs_make() {
         exit 1;
     fi
 
+    if [ -f "$CORTXFS_SRC/management/REST_doc.html" ]; then
+	    echo "Deleting Swagger Documentation"
+	    rm "$CORTXFS_SRC/management/REST_doc.html"
+    fi
+
+    sudo pip3 install PyYaml
+    echo "Generating Swagger Documentation"
+    python3 "$CORTXFS_SRC/management/swagger-yaml-to-html.py" < \
+    "$CORTXFS_SRC/management/openapi.yaml" > \
+    "$CORTXFS_SRC/management/REST_doc.html"
```

Comment:
It should be a part of a Makefile or something like that (a custom CMake taget):
```
doc: $DST/REST_doc.html

$DST/REST_doc.html: $SRC/openapi.yaml
  python3 "$CORTXFS_SRC/management/swagger-yaml-to-html.py"  $SRC/openapi.yaml $DST/REST_doc.html
```
Otherwise, this file will be generated every time we call "make". Moreover, it will be generated in the "source" directory which means we are breaking one of the most valuable features of our build system -- out-of-tree builds.